src/freenet/clients/http/ConnectionsToadlet.java

```diff
@@ -641,6 +637,30 @@ public void handleMethodPOST(URI uri, final HTTPRequest request, ToadletContext
 			String privateComment = null;
 			if(!isOpennet())
 				privateComment = request.getPartAsStringFailsafe("peerPrivateNote", 250).trim();
+
+			if (Boolean.parseBoolean(request.getPartAsStringFailsafe("peers-offers-files", 5))) {
+				File[] files = core.node.runDir().file("peers-offers").listFiles();
+				if (files != null && files.length > 0) {
+					StringBuilder peersOffersFilesContent = new StringBuilder();
+					for (final File file : files) {
+						if (file.isFile()) {
+							String filename = file.getName();
+							if (filename.contains(".") && "fref".equals(filename.substring(filename.lastIndexOf(".") + 1))) {
+								try (Scanner scanner = new Scanner(file)) {
+									peersOffersFilesContent.append(scanner.useDelimiter("\\A").next());
```

Comment:
`\A` is the boundary match for “beginning of the input,” what is that supposed to accomplish here?